src/collection.js

```diff
@@ -494,6 +494,23 @@ class Collection {
     return this
   }
 
+  /**
+   * Returns median of the current collection
+   *
+   * @param {Function} comparator
+   *
+   * @returns {Collection}
+   */
+  async median () {
+    const arr = Array.from(this.items.sort())
+    const mid = Math.floor(arr.length / 2)
+    if (arr.length % 2 !== 0) {
+      return arr[mid]
+    } else {
+      return (arr[mid] + arr[(mid - 1)]) / 2
+    }

```

Comment:
I'd love to refactor this if/else block to this:

```suggestion

    return arr.length % 2 !== 0
      ? return arr[mid]
      : (arr[mid] + arr[(mid - 1)]) / 2
```

Is that ok with you?

I like to keep an empty line before `if` and `return` statements :)

Reply:
I will make this change.

Reply:
Thank you

Reply:
Changes are done. Please review.